src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineCheck.java

```diff
@@ -70,6 +70,12 @@
  *   &lt;property name="format" value="\s+$"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
```

Comment:
Please make example of default config and make note that by default Check do nothing. No code sample is required for default.

Reply:
this is not done.
here is example on how to do this - https://github.com/checkstyle/checkstyle/blob/d62ad1a21b094f8ec115fd96cd593862d4ef8d32/src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineJavaCheck.java#L64

Reply:
this is mandatory !
please do.

Reply:
@Malintha1996 , this is still not done.
there should be config like `<module name="RegexpSingleline" \>` (no properties defined)

Reply:
> @Malintha1996 , this is still not done.
> there should be config like `<module name="RegexpSingleline" \>` (no properties defined)

done

Reply:
@Malintha1996 , please add text as 
https://github.com/checkstyle/checkstyle/blob/d62ad1a21b094f8ec115fd96cd593862d4ef8d32/src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineJavaCheck.java#L70-L74
as I requested originally, user need to see what is happening on default config usage.

Reply:
> @Malintha1996 , please add text as
> https://github.com/checkstyle/checkstyle/blob/d62ad1a21b094f8ec115fd96cd593862d4ef8d32/src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpSinglelineJavaCheck.java#L70-L74
> 
> 
> as I requested originally, user need to see what is happening on default config usage.

done

Reply:
you have to keep `To configure the check with default values`
please do this exactly as it is in provided example.
```
To configure the check with default values:
CONFIG
....explanation why we can not show a code ....
```